components/dropbox/dropbox.app.mjs

```diff
@@ -366,6 +367,7 @@ export default {
     },
     async restoreFile(args) {
       try {
+        console.log(args);
         const dpx = await this.sdk();
```

Comment:
Is this console.log() meant to be here, or was it left in from testing?

Reply:
Oops!